apis/rds/v1alpha1/zz_db_cluster.go

```diff
@@ -0,0 +1,508 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// DBClusterParameters defines the desired state of DBCluster
+type DBClusterParameters struct {
+	// Region is which region the DBCluster will be created.
+	// +kubebuilder:validation:Required
+	Region string `json:"region"`
+	// A list of Availability Zones (AZs) where instances in the DB cluster can
+	// be created. For information on AWS Regions and Availability Zones, see Choosing
+	// the Regions and Availability Zones (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.RegionsAndAvailabilityZones.html)
+	// in the Amazon Aurora User Guide.
+	AvailabilityZones []*string `json:"availabilityZones,omitempty"`
+	// The target backtrack window, in seconds. To disable backtracking, set this
+	// value to 0.
+	//
+	// Currently, Backtrack is only supported for Aurora MySQL DB clusters.
+	//
+	// Default: 0
+	//
+	// Constraints:
+	//
+	//    * If specified, this value must be set to a number from 0 to 259,200 (72
+	//    hours).
+	BacktrackWindow *int64 `json:"backtrackWindow,omitempty"`
+	// The number of days for which automated backups are retained.
+	//
+	// Default: 1
+	//
+	// Constraints:
+	//
+	//    * Must be a value from 1 to 35
+	BackupRetentionPeriod *int64 `json:"backupRetentionPeriod,omitempty"`
+	// A value that indicates that the DB cluster should be associated with the
+	// specified CharacterSet.
+	CharacterSetName *string `json:"characterSetName,omitempty"`
+	// A value that indicates whether to copy all tags from the DB cluster to snapshots
+	// of the DB cluster. The default is not to copy them.
+	CopyTagsToSnapshot *bool `json:"copyTagsToSnapshot,omitempty"`
+	// The DB cluster identifier. This parameter is stored as a lowercase string.
+	//
+	// Constraints:
+	//
+	//    * Must contain from 1 to 63 letters, numbers, or hyphens.
+	//
+	//    * First character must be a letter.
+	//
+	//    * Can't end with a hyphen or contain two consecutive hyphens.
+	//
+	// Example: my-cluster1
+	// +kubebuilder:validation:Required
+	DBClusterIdentifier *string `json:"dbClusterIdentifier"`
```

Comment:
This should be omitted by ACK. To do that, you need to add this field in every AWS call in `ignore.field_paths`. I think `DeleteDBCluster` is the missing call that you need to add.